openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -190,7 +190,7 @@ export function validateIdentifiers(data) {
     if (data.name === '' || data.name === '---') {
         return error('#id-errors', 'select-id', dataConfig['Please select an identifier.'])
     }
-    const label = $('#select-id').find(`option[value='${data.name}']`).html();
+    const label = $('#select-id').find("option[value='" + data.name + "']").html();
```

Comment:
This shouldn't be necessary, since it was already quoted :+1:

```suggestion
    const label = $('#select-id').find(`option[value='${data.name}']`).html();
```